config/resolve_bearer_token.go

```diff
@@ -54,20 +54,9 @@ func resolveBearerAuthTokenProviderChain(ctx context.Context, cfg *aws.Config, c
 
 	var provider smithybearer.TokenProvider
 
-	if sharedConfig.SSOSession != nil || (sharedConfig.SSORegion != "" && sharedConfig.SSOStartURL != "") {
-		ssoSession := sharedConfig.SSOSession
-		if ssoSession == nil {
-			// Fallback to legacy SSO session config parameters, if the
-			// sso-session section wasn't used.
-			ssoSession = &SSOSession{
-				Name:        sharedConfig.SSOStartURL,
-				SSORegion:   sharedConfig.SSORegion,
-				SSOStartURL: sharedConfig.SSOStartURL,
-			}
-		}
```

Comment:
Due to incorrect implementation before, removing the transformation of the legacy SSO session configuration is considered a break/fix and will need to be noted in the changelog / release notes.

Reply:
@jasdel im looking at the `changelog` announcement option:
```
-t <change-type> The change annotation type (release, feature, bugfix, dependency, announcement)
```
it looks like the announcement option would determine the change type. since my overall change is actually a new feature (token refresh support for sso), is there a way to tell the changelog tool that this change is a feature and to make such an announcement?

Reply:
The best way to do that is to create two different change log entries. a `feature` and a `announcement`. Its kind of tedious, but the tool doesn't allow manual specify version change strategy outside of the `change-type`.